### PR TITLE
News twitter feed

### PR DESCRIPTION
**Pull request by angfl97**
embedded twitter feed for LLNL_OpenSource on left side of news

addresses issue #103

### CODE REVIEW

File: news/index.md [hunk 1]
```diff
@@ -1,20 +1,43 @@
 ---
 title: News
-layout: default
+layout: news
 permalink: /news/
 ---
+{% raw %}
```
Comment (IanLee1521) [inline news/index.md]: Can we move this logic into the  `news.html` template? Or are we using that elsewhere?

File: news/index.md [hunk 2]
```diff
@@ -1,20 +1,43 @@
...
 
     {{ page.content }}
+
     </article>
   {% endfor %}
+  
+  <br />
+  <a class="btn btn-primary btn-block news" href="/news/archive/" role="button">See all news in the archive</a>
+  
+  <div class="twitterFeed-mobile" >
```
Comment (IanLee1521) [inline news/index.md]: This too I think should move into the template.

File: news/index.md (hunk 1 above)
Comment (angfl97) [reply news/index.md]: we can move out this logic into the template

File: news/index.md (hunk 2 above)
Comment (angfl97) [reply news/index.md]: done!

File: news/index.md (hunk 1 above)
Comment (angfl97) [reply news/index.md]: done!

**Comment by IanLee1521** [comment]
@hauten could you weigh in on this? Sample screenshots:

image

image

**Comment by hauten** [comment]
Looks good on the smaller screen but not the larger one. Would it be possible to move the feed inside the container that holds the news? OK if the news has to wrap around the feed, though in that case I would put the feed on the right side of the page.

I am on the fence about showing 2 vs 3 tweets. I've been trying to include an image with most tweets to catch readers' eyes, which makes 3 stacked tweets take up a lot of space. Maybe that's OK on a page that holds 20 news items of varying lengths.

**Comment by angfl97** [comment]
News wrapped around the twitter feed, here's what it looks like with 3 tweets

Screenshot-2019-7-10 News

and with 2 tweets
Screenshot-2019-7-10 News(1)

also this is how it would scale on a smaller screen
Screenshot-2019-7-10 News(2)

**Comment by hauten** [comment]
Nice! Let's do 2 tweets so the "follow" button stays above the fold.

**Comment by angfl97** [comment]
2 tweets it is

**Pull request merged**